### DISCUSSION TITLE
Option to autoexpand product variatons

### DISCUSSION THREAD

**Discussion by yvovandoorn** [pretix/pretix#2937] [Ideas] [answered]
Use case: A product SKU has two variations: 'early-bird' and 'standard'. The 'early-bird' variation is a discounted version of the 'standard' variation. Once the 'early-bird' variation expires, the 'standard' variation takes over. 

Currently the UI, both the widget and shop, requires someone to click 'show variations', yet there is only 'one' variation. 

<img width="993" alt="webshop" src="https://user-images.githubusercontent.com/2694655/205064663-9fcb885e-62e1-4063-b1ff-2cbc9754c3c0.png">
<img width="1152" alt="widget" src="https://user-images.githubusercontent.com/2694655/205064667-f7e9b2b8-983b-405d-a398-fc3d8fcf9bc7.png">

What I would like is a way to 'auto-expand' all variations as a setting. So instead of the user being asked to click 'see variations'. From an UX perspective this would be friendlier when there are less variations (like in this case, one) available.

**Comment by raphaelm** [comment, marked as answer]
The setting already exists :) 
![20221201_145151](https://user-images.githubusercontent.com/64280/205070199-e2abc288-508a-4971-bf10-ffa3b502a46b.png)

**Comment by yvovandoorn** [comment]
Well that is even better. Thank you @raphaelm :-)
